sklearn/calibration.py

```diff
@@ -87,6 +87,12 @@ class CalibratedClassifierCV(ClassifierMixin, MetaEstimatorMixin, BaseEstimator)
     The calibration is based on the :term:`decision_function` method of the
     `estimator` if it exists, else on :term:`predict_proba`.
 
+    For examples of usage, see:
+    :ref:`sphx_glr_auto_examples_calibration_plot_calibration.py`.
+    :ref:`sphx_glr_auto_examples_calibration_plot_calibration_curve.py`.
+    :ref:`sphx_glr_auto_examples_calibration_plot_calibration_multiclass.py`.
```

Comment:
Same here, I'd move this to the end of the "Examples" section of the docstring for consistency

Reply:
DONE.